resources/profiles/Anker/machine/Anker M5 All-Metal 0.4 nozzle.json

```diff
@@ -0,0 +1,37 @@
+{
+    "type": "machine",
+    "name": "Anker M5 All-Metal 0.4 nozzle",
+    "inherits": "fdm_marlin_common",
+    "from": "system",
+    "setting_id": "GM002",
+    "instantiation": "true",
+    "nozzle_diameter": [
+        "0.4"
+    ],
+    "printer_model": "Anker M5 All-Metal Hot End",
```

Comment:
usually, it's not necessary to create such tiny variant.
It would cause extra confusion for users.
I suggest we keep the old printer model profile

Reply:
So, in this case, it's actually causing confusion by not having it...that's why I wanted to add this change in the first place. As it happens right now, if someone using the all-metal variant of the hot end tried to print PLA with the default profile, it will actually cause a clog...Separating it out made the most sense in my mind at least.